lib/Service/SynchronizationService.php

```diff
@@ -383,6 +421,7 @@ public function getAllObjectsFromApi(Synchronization $synchronization, ?bool $is
         ];
 
         // Make the initial API call
+		// @TODO: method is now fixed to GET, but could end up in configuration.
```

Comment:
Just a reminder that we should avoid doing this (even though I do it myself as well 😝)
Maybe create an issue for this, or mention it in another existing issue?

Reply:
We should create an issue for this. I refined with @rubenvdlinde that I would put in these TODOs

Reply:
Please show me the new issue created for this

Reply:
https://conduction.atlassian.net/browse/CONNECTOR-152?atlOrigin=eyJpIjoiZmEwYTA0NGE4NjlkNGEzODk3NTUxNmIzYTRlMTUwZDMiLCJwIjoiaiJ9